ulib/ulibfs.fsproj

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I had to move ulibfs project one level up, as running fstar extraction from within ulib/fs directory was causing issues with "prims.fs being unknown extension". FStar was suggesting to add --MLish flag, but it didn't work.